### ISSUE TITLE
Unix timestamp serializer for subsecond accuracy

### ISSUE THREAD

**Issue by mitsuhiko** [chronotope/chrono#227]
I currently need to use a custom serializer/deserializer to format my timestamps into f64 values for JSON because the builtin serializer/deserializer for serde can only do seconds accuracy. This seems common enough of a case that shipping this feature seems useful.

**Comment by quodlibetor** [comment]
Agreed! If you send a PR I'll happily merge it. Doing something like [the `serde::ts_seconds` module](https://github.com/chronotope/chrono/blob/c9609ea6c9dc1/src/naive/datetime.rs#L1640-L1747) is I think what this should look like, although I believe I've seen slightly more pleasant serde patterns than what I initially wrote in that module.
